### PR TITLE
As

### PR DESCRIPTION
**Pull request by LazavikouArtsiom**

### CODE REVIEW

File: jsonconverter.py [hunk 1]
```diff
@@ -1,81 +1,250 @@
...
+        except FileExistsError:
+            print('File is already exist. Choose another one')
+            cls._create_file(caused)
+        return _file
+
+
+class CsvToJson(Converter):
+    def __new__(cls):
+        return cls
+
```
Comment (MaksimLion) [inline jsonconverter.py]: Are you sure that it is right implementation of `__new__`?

**Comment by MaksimLion** [comment]
Also, you forgot about point with multiple values

File: jsonconverter.py [hunk 2]
```diff
@@ -1,81 +1,250 @@
...
+        
+        _file = input('Enter name of file to write :').split(".")[0]
+        if caused == "csv":  #if it's caused from CsvToJson - create .json file
+            _file += ".json"
+        else:
+            _file += ".csv"
+        try:
+            with FileManager(_file, 'x'):
+                pass
+        except FileExistsError:
```
[the remark this answers is not in the document]
Comment (LazavikouArtsiom) [reply jsonconverter.py]: image

[the remark this answers is not in the document]
Comment (LazavikouArtsiom) [reply jsonconverter.py]: But it works!

**Pull request merged**